www/analytics.js

```diff
@@ -60,7 +60,16 @@ UniversalAnalyticsPlugin.prototype.trackMetric = function(key, value, success, e
       throw Error("key must be a valid integer or string formatted integer");
     }
   }
-  cordova.exec(success, error, 'UniversalAnalytics', 'trackMetric', [numberKey, value]);
+
+  // as google analytics sdk expects value of type string and internally converts it to type number,
+  // this plugin expects a number as value.
+  // all platform implementations expect value of type string,
+  // so value has to be casted to string type before calling the platform plugin.
+  var stringValue = value;
```

Comment:
Yes and no.

Google Analytics SDK for iOS seems to expect a NSString type for value.
As [in the docs](https://developers.google.com/analytics/devguides/collection/ios/v3/customdimsmets) example:
```ObjectiveC
// Set the custom metric to be incremented by 5 using its index.
[tracker set:[GAIFields customMetricForIndex:1]
       value:[[NSNumber numberWithInt:5] stringValue]];
```

But you are right, in the Android implementation it is always float.
So i should add an additional fix in my PR.

The change in www/analytics is needed anyway, because all implementations of the platform trackMetric implementations were designed to accept value as string. (and to convert it if necessery, like on Android..)
I made this type casting to be backwards compatible this way and not to be forced to touch iOS implemention, which i can not test in my environment..

I'll hopefully add another commit to PR tomorrow.
I have to test the change to Android implementation and dig a bit deeper in the Windows native UWP implementation, as it seems value is defined as c++ long long type instead of float or double..